### PR TITLE
Bulgarian Translation 11

### PR DESCRIPTION
**Pull request by Effektus**
71% translated

### CODE REVIEW

**Comment by zenseii** [comment]
Hi @Effektus, could you solve the merge conflict? I belive you found out how to, right?

**Review by zenseii — APPROVED** [review]
Good job @Effektus! We're getting closer to the finish line every time.

File: files/lang/bg.po [hunk 1]
```diff
@@ -4081,41 +4080,40 @@ msgid "Game saved successfully."
...
 "Are you sure you want to load a new game? (Your current game will be lost.)"
 msgstr ""
+"Сигурен ли си, че искаш да заредиш нова игра? (Текущата ти игра ще бъде "
+"загубена.)"
 
 msgid "Try looking on land!!!"
-msgstr ""
-
-msgid ""
-"Searching for the Ultimate Artifact is fruitless. Your hero could not carry "
```
Comment (zenseii) [inline files/lang/bg.po:4094]: Hi @Effektus.

This string needs to be here since it was newly added.

**Review by zenseii — APPROVED** [review]

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Effektus , thanks a lot for keeping the translation up to date!

**Pull request merged**